src/annotations/_AnnotationManager.js

```diff
@@ -1,51 +1,62 @@
-import dcmjs from 'dcmjs'
+import dcmjs from "dcmjs";
+import Feature from "ol/Feature";
 
-import _MarkupManager from './markups/_MarkupManager'
+import _MarkupManager from "./markups/_MarkupManager";
```

Comment:
Could you please revert these stylistic changes? They complicate the code review and are not in compliance with the coding style of the library (see [Coding Style section in CONTRIBUTING](https://github.com/MGHComputationalPathology/dicom-microscopy-viewer/blob/master/CONTRIBUTING.md#coding-style)).